netcontrold/lib/dataif.py

```diff
@@ -805,16 +805,17 @@ def get_pmd_rxqs(pmd_map):
         elif re.match(r'\s.*port: .*', line):
             # From this line, we retrieve cpu usage of rxq.
             linesre = re.search(r'\s.*port:\s([A-Za-z0-9_-]+)\s*'
-                                r'queue-id:\s*(\d+)\s*'
+                                r'queue-id:\s*(\d+)\s*(?=\((enabled)\))?.*'
                                 r'pmd usage:\s*(\d+|NOT AVAIL)\s*?',
                                 line)
 
             pname = linesre.groups()[0]
             qid = int(linesre.groups()[1])
+            enabled_flag = linesre.groups()[2]
             try:
-                qcpu = int(linesre.groups()[2])
+                qcpu = int(linesre.groups()[3])
             except ValueError:
-                qcpu = linesre.groups()[2]
+                qcpu = linesre.groups()[3]
```

Comment:
I didn't change it. It is like that because.. in try: code checks for qcpu which is int type.. if it receives valuerror because of 'NOT AVAIL' .. then not casting is required..

Reply:
Good catch.